docs/schedule.md

```diff
@@ -473,8 +473,9 @@ All times listed below for the main virtual event are in [PDT (UTC-7)](https://w
     <tr>
       <td>11:00AM - 11:45AM<br>PDT</td>
       <td>
-        <strong>Neural networks in R</strong>
-        <br>
+        <strong>Neural networks in R</strong><br>
+        <a href="https://github.com/oceanhackweek/ohw-tutorials/tree/master/r-keras-test">R Markdown</a>
```

Comment:
Hmm, it looks like this may be an early version. It's not what he used today, which was https://github.com/oceanhackweek/ohw-tutorials/tree/master/deep-learning. I've asked Johnathan to clarify, via Slack.

Reply:
Change `<a href="https://github.com/oceanhackweek/ohw-tutorials/tree/master/r-keras-test">R Markdown</a>` to 
`<a href="https://github.com/oceanhackweek/ohw-tutorials/tree/master/deep-learning">Presentation and R Jupyter notebooks</a>`